x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/components/config_form.tsx

```diff
@@ -170,49 +169,34 @@ export const AgentConfigForm: React.FunctionComponent<Props> = ({
           />
         }
       >
-        <EuiSwitch
-          showLabel={true}
-          label={
+        <EuiFormRow
+          fullWidth
+          error={touchedFields.namespace && validation.namespace ? validation.namespace : null}
+          isInvalid={Boolean(touchedFields.namespace && validation.namespace)}
+          helpText={
             <FormattedMessage
-              id="xpack.ingestManager.agentConfigForm.namespaceUseDefaultsFieldLabel"
-              defaultMessage="Use default namespace"
+              id="xpack.ingestManager.agentConfigForm.namespaceFieldHelpText"
+              defaultMessage="Leave empty if you do not want a namespace."
```

Comment:
I don't think this is correct. There is always a namespace called `default`.

Reply:
@ruflin That's new to me, I thought we allowed no namespaces. So if the user leaves this input empty, we add `default` to the agent config behind the scenes? Is that the same for the namespace field when creating a data source?

Reply:
Yes. There must always be a namespace and agent will fill it in if it is not provided. @ph Just to confirm.

I'm still on the fence if we need this setting here at all on the config level. If we do, we must probably better describe what it is for, meaning telling the user this is about where the data is shipped.

Reply:
That might be something to look at on the agent (and/or EPM?) side. I've used agent configs with no namespaces that end up creating indices without a namespace suffix like `metrics-system.core-`.

I'll confirm this behavior and create follow up issues where we can discuss how to handle empty namespaces. I think that would be separate from this PR as the PR is UI-only change. The "Leave empty if you do not want a namespace" copy is correct with regards to how we currently handle things, even though it's not the behavior we want.

Reply:
qq